src/console/Microsoft.Health.Fhir.Ingest.Console.csproj

```diff
@@ -7,6 +7,17 @@
 	<PropertyGroup>
 		<IsPackable>true</IsPackable>
 	</PropertyGroup>
+	<ItemGroup>
+	  <Compile Remove="Microsoft.Health.Fhir.Ingest.Console.Common\**" />
+	  <Compile Remove="Microsoft.Health.Fhir.Ingest.Console.FhirTransformation\**" />

```

Comment:
I believe this is related to how the namespaces are named. Whenever compiling Microsoft.Health.Fhir.Ingest.Console some of the descendants were getting pulled in, so this was added this to prevent it. we will be deleting Microsoft.Health.Fhir.Ingest.Console.csproj anyway so didn't spend much time trying to find an ideal solution.

Reply:
Okay, I didn't think we had any embedded resources so that part was confusing to me.